tmgmt_thebigword.module

```diff
@@ -65,3 +65,32 @@ function tmgmt_thebigword_form_tmgmt_job_item_edit_form_alter(&$form, FormStateI
     }
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function tmgmt_thebigword_form_tmgmt_settings_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  $config = \Drupal::configFactory()->get('tmgmt_thebigword.settings');
+
+  $form['thebigword'] = [
+    '#type' => 'details',
+    '#title' => t('thebigword Connector for Drupal'),
+    '#open' => TRUE,
+  ];
+  $form['thebigword']['debug'] = [
+    '#type' => 'checkbox',
+    '#title' => t('Debug mode'),
+    '#default_value' => $config->get('debug'),
+    '#description' => t('Check it to enable the debugging mode.'),
```

Comment:
Enabling this will log all communication with thebigword API. Should not be enabled on production.